com.unity.netcode.gameobjects/Components/NetworkTransform.cs

```diff
@@ -15,9 +15,11 @@ namespace Unity.Netcode.Components
     [DefaultExecutionOrder(100000)] // this is needed to catch the update time after the transform was updated by user scripts
     public class NetworkTransform : NetworkBehaviour
     {
-        public const float PositionThresholdDefault = .001f;
-        public const float RotAngleThresholdDefault = .01f;
-        public const float ScaleThresholdDefault = .01f;
+        public const float ThresholdMinimum = 0.001f;
+        public const float PositionThresholdDefault = 0.001f;
+        public const float RotAngleThresholdDefault = 0.01f;
+        public const float ScaleThresholdDefault = 0.01f;
```

Comment:
I was wondering about using [`Mathf.Epsilon`](https://docs.unity3d.com/ScriptReference/Mathf.Epsilon.html) for these... any thoughts?

Reply:
I think that could encourage a flawed design for detecting a change in a value to update over a network.  Setting it to something like the Epsilon value ( 1.401298E-45) is such a small value that it is about the same as setting it to zero which can lead to scenarios where you are sending changes close to every frame due to minor jitter in things like rotation and/or moving along a slope with gravity applied.

Of course, I did think about scenarios where someone uses something like 0.001 as their game's 1 distance unit which would make detecting changes to that difficult...so setting the "minimum threshold" to a very low value could result in sending updates more frequently than desired (by default) but setting it too high could cause issues where the users world scale is not the default unity world scale (i.e. 1.0 tends to be the default in Unity).
(shrug)

I can see pro's and con's both ways.  I picked 1/1000th of a "default world space unit" and 1/100th of a Euler degree for rotation....but maybe I just remove the restriction on scale and position values and delete ThresholdMinimum?
We want to keep the rotation thresholds as those make sense no matter what the user's world scale.

Reply:
I think 0.001f as defaults is fine but making 0.001f the threshold minimum might be a little limiting. If I disable gravity or make my rigidbody kinematic, it wouldn't move at all, not even epsilon and that _shouldn't_ cause a re-replication since `oldValue - newValue > epsilon` would be `false`, right?
also I feel like we could have tests just testing these ranges too.
so if I were to sum it up, I think we should keep sensible defaults as 0.01f & 0.001f but make epsilon the min value for the ranges.